pwa/src/templates/templateParts/mappingForm/MappingFormTemplate.tsx

```diff
@@ -22,15 +27,32 @@ export const MappingFormTemplate: React.FC<MappingFormTemplateProps> = ({ mappin
   const [_mapping, setMapping] = React.useState<any[]>([]);
   const [unset, setUnset] = React.useState<any[]>([]);
   const [cast, setCast] = React.useState<any[]>([]);
+  const [isOpenMapping, setIsOpenMapping] = React.useState<boolean>(false);
+  const [isOpenUnset, setIsOpenUnset] = React.useState<boolean>(false);
+  const [isOpenCast, setIsOpenCast] = React.useState<boolean>(false);
```

Comment:
We can open the accordions by default when creating a mapping, like so: `React.useState....(!mapping)` (dummy code)